nixos/modules/services/misc/autobrr.nix

```diff
@@ -0,0 +1,70 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
consider `inherits` where you find yourself using `lib.*` excessively.
[this](https://nix.dev/guides/best-practices.html#with-scopes)  nix.dev page on best practices with `with` may be helpful
an issue (#208242) has been made to track inappropriate uses of `with` such as this.